### PR TITLE
More ckeditor cleanup

### PR DESCRIPTION
**Pull request by rshipp**
### Description

More deleting, and some refactoring around "fileuploadquestion" and "radioquestion"

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by sadleb — COMMENTED** [review]
Defer to @jessouyangbraven on this. I’m not well versed enough on CKEditor to be able to provide a second set of eyes.

File: app/javascript/ckeditor/contentcommonediting.js [hunk 1]
```diff
@@ -6,11 +6,10 @@ import List from '@ckeditor/ckeditor5-list/src/list';
 import Table from '@ckeditor/ckeditor5-table/src/table';
 import RetainedData from './retaineddata';
 import InsertTextInputCommand from './inserttextinputcommand';
-import InsertFileUploadQuestionCommand from './insertfileuploadquestioncommand';
+import InsertFileUploadCommand from './insertfileuploadcommand';
 import InsertTextAreaQuestionCommand from './inserttextareaquestioncommand';
 import InsertTextAreaCommand from './inserttextareacommand';
 import SetAttributesCommand from './setattributescommand';
-import { ALLOWED_ATTRIBUTES, filterAllowedAttributes } from './customelementattributepreservation';
```
Comment (jessouyangbraven) [inline app/javascript/ckeditor/contentcommonediting.js:13]: Does this mean we're getting rid of retained data?!

**Review by jessouyangbraven — APPROVED** [review]
LGTM, though I am a bit rusty on the CKE stuff.

File: app/javascript/ckeditor/contentcommonediting.js (hunk 1 above)
Comment (rshipp) [reply app/javascript/ckeditor/contentcommonediting.js:13]: yeah we never had a discussion about this but I assume we're just using `name` from here on out, I can't think of any reason we need to stay backwards compatible

**Pull request merged**